### DISCUSSION TITLE
Error while downloading results from ASReview LAB?

### DISCUSSION THREAD

**Discussion by SagevdBrand** [asreview/asreview#544] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
While downloading the results of a short screening procedure of the cord-19 dataset, I received the following error:
"ValueError: The state file does not correspond to the given data file, please use another state file or dataset."

Any suggestions on how to circumvent this error?

**Comment by J535D165** [comment, marked as answer]
Did you select multiple datasets in the datasets upload step? You can look this up in the project folder at `~/.asreview`. Do you see multiple datasets in the data folder?

I think the first dataset took a while to download and was therefore added after the second dataset was selected. Therefore they are interchanged. I think @terrymyc partially fixed this in the benchmark datasets tab. 

Can you make an issue in the issue tracker as well?

**Reply by SagevdBrand** [reply]
Ah I probably did! But I cleaned all the projects as proposed [here](https://asreview.readthedocs.io/en/latest/intro/installation.html?highlight=project%20folder#remove-temporary-files).  Which also solved the issue! So the link here is actually already the solution.

Will make an issue too!  Thanks for the help 💪
